docs(ipfs): typography and more explaination (#1722)
scaleway/docs-content: labs/ipfs/api-cli/ipfs-get-content.mdx

---
meta:
  title: How to get your pinned content
  description: How to get your pinned content.
content:
  h1: How to get your pinned content
  paragraph: How to get your pinned content.
tags: ipfs pinning storage ipfs-storage volume desktop labs desktop web3 gateway kubo
dates:
  validation: 2023-05-25
  posted: 2023-05-25
categories:
  - labs
  - storage
  - ipfs-storage
  - pinning
  - ipfs-pinning
---

## Get your content from Kubo client

Once your content is pinned on our service, you can fetch it via any other [Kubo client](https://docs.ipfs.tech/install/command-line/). It is Linux/Max/Windows compatible.

First, make sure your client is running:

```
ipfs init && ipfs daemon
```

> then, just do as follow:

Reviewer: ```suggestion
Then, run the following command. 
```

```
ipfs get <YOUR_PINNED_CID> > myFile
```

> Your content will be stored into myFile.

Reviewer: ```suggestion
Your content will be stored in `myFile`.
```

> ## Get your content from a HTTP gateway

Reviewer: ```suggestion
## Get your content from an HTTP gateway
```

If you don't want to bother you with a local client, you can fetch public content via an IPFS gateway. Up to you, to use your web browser or curl.

```
curl https://ipfs.io/ipfs/<YOUR_PINNED_CID>
```

> ## Get your content from the go client

Reviewer 2: ```suggestion
## Get your content from the Go client
```

> If you plan to fetch your pinned content via a go program, we recommand you to fetch the official [go-ipfs-api package](https://github.com/ipfs/go-ipfs-api).

Reviewer 2: ```suggestion
If you plan to fetch your pinned content via a Go program, we recommend to use the official [go-ipfs-api package](https://github.com/ipfs/go-ipfs-api).
```

> In that case, you need to run an IPFS node (like Kubo) to perform operations with the IPFS network.

Reviewer: ```suggestion
In this case, you need to run an IPFS node (like Kubo) to perform operations with the IPFS network.
```
